en/getting-started/monitor-cisco-router-with-snmp

```diff
@@ -0,0 +1,75 @@
+---
+id: monitor-cisco-router-with-snmp
+title: Monitor your first Cisco router
+---
+
+### Monitoring a Cisco Router with SNMP
+
+In this tutorial, we're assuming that your Centreon platform is installed and running well, and that you have at least a [Centreon IT 100 Edition](IT100.html) that provides Centreon Plugin Packs (your [license](../administration/licenses.html) is already set up).
+
+Your Cisco router will be monitored using the [Cisco standard](../integrations/plugin-packs/procedures/network-cisco-standard-snmp.html). (More about Plugin Packs [here](../monitoring/pluginpacks.html)). 
+
+## Prerequisites
+
+### On the cisco router you want to monitor
+
+The first step is to activate and configure an SNMP service on your Cisco device.
+Please refer to the documentation of your Cisco device to know how to configure the SNMP service. You have an example on the monitoring procedure of [Cisco standard](../integrations/plugin-packs/procedures/network-cisco-standard-snmp.html).
+
+### Network flow
+
+The target device must be reachable from the Centreon Poller on the UDP/161 SNMP port.
+
+### On the poller
+
+1. Connect to your poller in SSH and install the Cisco standard plugin (see the [monitoring procedure for the **Cisco standard** Plugin Pack](../integrations/plugin-packs/procedures/network-cisco-standard-snmp.html) for more information):
+
+   ```shell
+   yum install centreon-plugin-Network-Cisco-Standard-Snmp
+   ```
+
+2. In the web interface, go to **Configuration > Plugin Packs**, search and install the **Cisco standard** Plugin Pack:
+
+   ![image](../assets/getting-started/quick_start_cisco_0.gif)
+
+Go to the **Configuration > Hosts > Hosts** menu and click on **Add**:
+
+![image](../assets/getting-started/quick_start_cisco_1.gif)
+
+Fill in the following information:
+
+* The name of the router (1)
+* A description of the router (2)
+* The IP address (3)
+* The SNMP version and community (4)
+* Select the monitoring poller (keep "Central" if you have no other poller) (5)

```

Comment:
```suggestion
* Select the poller that will monitor the device (keep "Central" if you have no other poller) (5)

```

Reply:
Same as above, I've taken what we have writen for linux server

Reply:
Maybe we could improve the Linux topic too, then...